charts/tfy-gpu-operator/values.yaml

```diff
@@ -339,6 +332,45 @@ gcp-gke-standard-dcgm-exporter:
     - name: nvidia-config
       mountPath: /etc/nvidia
 
+## @section azure-aks-dcgm-exporter Configuration for the Azure AKS Nvidia Device Plugin. This section will only be used when clusterType.azureAks is set to true.
+##
+azure-aks-nvidia-device-plugin:
+  namespaceOverride: tfy-gpu-operator
+  updateStrategy:
+    type: OnDelete
+  podAnnotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ""
+  compatWithCPUManager: true
+  deviceListStrategy: volume-mounts
+  deviceIDStrategy: index
```

Comment:
These are new additions which were not enabled before, will check if they make a difference